src/runtime_src/core/edge/user/aie/graph_object.cpp

```diff
@@ -39,6 +39,9 @@ graph_object::graph_object(ZYNQ::shim* shim, const xrt::uuid& uuid , const char*
 
   /* Initialize graph rtp metadata */
   rtps = xrt_core::edge::aie::get_rtp(device.get(), graph_config.id, m_hwctx);
+  /* Initialize graph shared buffer config */
+  shared_buffer_configs = xrt_core::edge::aie::get_shared_buffers(device.get());
```

Comment:
As discussed offline, These shared_buffer_configs should be per graph_id and should support hw_context also.
Please make that change